java/src/test/java/ai/rapids/cudf/TimestampColumnVectorTest.java

```diff
@@ -350,6 +413,37 @@ public void testAddMonths() {
     }
   }
 
+  @Test
+  public void testAddMonthsScalar() {
+    long[] EXPECTED = new long[]{
+        -129290327762L,   //'1965-11-26 14:01:12.238' Friday
+        1533384000115L,   //'2018-08-04 12:00:00.115' Saturday
+        1677310332929L,   //'2023-03-25 07:32:12.929' Saturday
+        -129290327762L,   //'1965-12-26 14:01:12.238' Sunday
+        1533384000115L};  //'2018-09-04 12:00:00.115' Tuesday
+    try (ColumnVector timestampColumnVector = ColumnVector.timestampMilliSecondsFromLongs(TIMES_MS);
+         Scalar months = Scalar.fromShort((short)1);
+         ColumnVector result = timestampColumnVector.addCalendricalMonths(months);
+         ColumnVector expected = ColumnVector.timestampMilliSecondsFromLongs(EXPECTED)) {
+      assertColumnsAreEqual(expected, result);
+    }
+  }
+
+  @Test
+  public void testDaysInMonth() {
+    short[] EXPECTED = new short[]{
+        31,   //'1965-10-26 14:01:12.238' Tuesday
+        31,   //'2018-07-04 12:00:00.115' Wednesday
+        31,   //'2023-01-25 07:32:12.929' Wednesday
+        31,   //'1965-10-26 14:01:12.238' Tuesday
+        31};  //'2018-07-04 12:00:00.115' Wednesday
```

Comment:
Silly question:  Ought we to consider adding test rows where the month has fewer than 31 days?  To avoid impact on other tests, we might consider using a test input vector local to this test.

Reply:
Yes and no. The tests here are to make sure that we have wired things up correctly. It is not really intended to verify that CUDF is doing what it should do. CUDF should have tested that already.  I am fine with adding more tests, especially for this case where it might be a bit odd.

Reply:
Not necessary.  This is mostly just a ping-check.